src/libsyntax/diagnostics/plugin.rs

```diff
@@ -20,6 +20,9 @@ use parse::token;
 use ptr::P;
 use util::small_vector::SmallVector;
 
+// Maximum width of any line in an extended error description (inclusive).
+static MAX_DESCRIPTION_WIDTH: usize = 80;
```

Comment:
This probably makes more sense as a `const` (we care about the value itself, not the memory location).